llvm/include/llvm/ADT/AddressRanges.h

```diff
@@ -22,14 +22,16 @@ namespace llvm {
 class AddressRange {
 public:
   AddressRange() {}
-  AddressRange(uint64_t S, uint64_t E) : Start(S), End(E) {
+  AddressRange(const uint64_t S, const uint64_t E) : Start(S), End(E) {
```

Comment:
We do not add const to non-reference/-pointer function arguments

Reply:
Genuinely curious, what's the reasoning for this?

Reply:
In C++ it's weird because it's not a part of function signature. Since the rest of the project doesn't use it, we shouldn't adopt this style here either

Reply:
From what I read on certain sources the top-level `const`-qualification matters in definitions but not declarations, and since this is a definition that's where my confusion arises. Also noted in [[readability-avoid-const-params-in-decls]](https://releases.llvm.org/20.1.0/tools/clang/tools/extra/docs/clang-tidy/checks/readability/avoid-const-params-in-decls.html).

However I agree with the style part.

Reply:
I agree, I'll revert these changes.